### ISSUE TITLE
Archlinux AUR

### ISSUE THREAD

**Issue by mnivoliez** [relamptk/vr-streaming-overlay#14]
Hello me again,

I was wondering if your program could not be distributed in archlinux aur depot?

**Comment by beniwtv** [comment]
Yes, it can and probably will at some point when I get the time to make a PKGBUILD for it

**Comment by mnivoliez** [comment]
To be honest I'm already looking into it, once I got a a good grip on a pkgbuild, may I post it here for you to see if it fits?

**Comment by beniwtv** [comment]
Sure, any help is welcome :)

**Comment by mnivoliez** [comment]
Ok, I made one. It is basic but could be a start. Where do you want to put it?
Should I make a merge request to put it inside the repo or should I drop it in here?

**Comment by beniwtv** [comment]
Yeah if you could attach it here would be great :)

**Comment by mnivoliez** [comment]
Here we go!
I put it inside a gist for now as I can't just drag'n drop the file here:
[PKGBUILD](https://gist.github.com/mnivoliez/af12bbef9fe3b70000e1c8ce2f0ca130)

Please, not that only the release is covered by this PKGBUILD, I'll try to make another one for the git version.

**Comment by beniwtv** [comment]
That looks good already!

ok I think we have a few things to solve here before we can push it to AUR:
1) The package name for my next release needs to match "vr-streaming-overlay", not "vroverlay", so the AUR package can be called "vr-streaming-overlay" for consistency
2) Description needs to be reworked a bit - will update it in README.md
3) I need to include icon.png and a .desktop file in the repo and release export, so an icon can be installed for the user to launch it comfortably

A vr-streaming-overlay-git AUR package would be really nice too, basically you'd have to:
1) Grab the current Godot 3.2.1 server build and use --export parameter to export the project you pulled from GIT, then package that export up similarily to the non-git version

I will try to make those changes and a 0.0.4-alpha this weekend, which is a long weekend for me so I have more time \o/

**Comment by mnivoliez** [comment]
I also got a long week end (monday is a day off where I live), so I'll work on those changes. Should we create repos for pkgbuild or something? I never done one before so I'm kind of discovering all that.

**Comment by beniwtv** [comment]
I still have to read up on AUR what is the procedures there for the PKGBUILDs. Ideally I'd liek to have it in this repo.

**Comment by beniwtv** [comment]
I made good progress on this; I now download the source directly from GIT and "compile" it with Godot, this saves us from making a -bin AUR package.

I also registered with the AUR and the PKGBUILD is almost modified, however ran into #11 myself - I made a fix for godot_openvr that needs to go through first.

Actually seems to be an issue on Godot's side https://github.com/godotengine/godot/issues/36781

**Comment by mnivoliez** [comment]
Hi, have you the PKGBuild somewhere?
Have you decided where to put it?

**Comment by beniwtv** [comment]
Yes we will keep a copy here in the repo, and I wil upload them to AUR by Git (that seems to be the way to do it). Still working on all that.

**Comment by tingvarsson** [comment]
Hi, are there still plans for maintaining this project, and doing an AUR build as suggested here?

**Comment by beniwtv** [comment]
Yes, lately due to COVID I have not had much time for it, but holidays are coming up so expect some updates :)

**Comment by tingvarsson** [comment]
@beniwtv Sweet to hear!
I understood that the latest release is broken and I should run latest master?
Btw, are there ANY alternatives for twitch chat on linux right now? would be interesting to test, but havent found any.

**Comment by beniwtv** [comment]
There is xrdesktop, which you can use to get your desktop windows into your VR world, some streamers use that. You could use that to beam pop-out Twitch chat windows in.
